tamr_client/categorization/project.py

```diff
@@ -0,0 +1,58 @@
+from tamr_client._types import (
+    CategorizationProject,
+    Dataset,
+    Instance,
+    JsonDict,
+    Session,
+    URL,
+)
+from tamr_client.dataset import _dataset, unified
+
+
+def _from_json(url: URL, data: JsonDict) -> CategorizationProject:
+    """Make Categorization project from JSON data (deserialize)
+
+    Args:
+        url: Project URL
+        data: Project JSON data from Tamr server
+    """
+    return CategorizationProject(
+        url, name=data["name"], description=data.get("description")
+    )
+
+
+def manual_labels(
+    session: Session, instance: Instance, project: CategorizationProject
+) -> Dataset:
+    """
+    Get manual labels from a Categorization project
```

Comment:
nit: summary line should be on the same line as leading `"""` with a newline after